events/cognito.go

```diff
@@ -106,3 +106,24 @@ type GroupConfiguration struct {
 	IAMRolesToOverride []string `json:"iamRolesToOverride"`
 	PreferredRole      *string  `json:"preferredRole"`
 }
+
+// CognitoEventUserPoolsCustomMessage is sent by AWS Cognito User Pools before a verification or MFA message is sent,
+// allowing a user to customize the message dynamically.
+type CognitoEventUserPoolsCustomMessage struct {
+	CognitoEventUserPoolsHeader
+	Request  CognitoEventUserPoolsCustomMessageRequest  `json:"request"`
+	Response CognitoEventUserPoolsCustomMessageResponse `json:"response"`
+}
+
+// CognitoEventUserPoolsCustomMessageRequest contains the request portion of a CustomMessage event
+type CognitoEventUserPoolsCustomMessageRequest struct {
+	UserAttributes map[string]bool `json:"userAttributes"`
```

Comment:
shouldn't this be `map[string]string`? https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-message.html#cognito-user-pools-lambda-trigger-syntax-custom-message

Reply:
ugh, I see the sample now - This appears to need to be `map[string]interface{}`

Reply:
yes, you're right it should be interface{} instead of bool I will change it. Great thanks for review

Reply:
done